de-lib/src/main/java/org/iplantc/de/apps/widgets/client/presenter/AppLaunchPresenterImpl.java

```diff
@@ -300,13 +303,30 @@ private HandlerManager createHandlerManager() {
         return new HandlerManager(this);
     }
 
+    private boolean hasConcurrencyError(Throwable caught) {
+        String errMessage = caught.getMessage();
+        return errMessage.matches("\\S+ is already running \\d+ or more concurrent jobs\\n");
```

Comment:
Right now, this seems to be the only message returned to the UI.  If the error response gets updated to provide a predefined key, this can be updated to check for that instead.